quadriga/src/main/java/edu/asu/diging/quadriga/domain/elements/Term.java

```diff
@@ -0,0 +1,105 @@
+package edu.asu.diging.quadriga.domain.elements;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * This file contains the definition of Term class.
+ *
+ */
+@XmlRootElement
+public class Term extends Element {
+
+    Long graphId;
```

Comment:
what are these graph ids used for? I don't think they are necessary as one graph should map to one document in mongodb?